### PR TITLE
Artifacts for .NET 9 200 Preview SDK Release

### PR DESCRIPTION
**Pull request by rbhanda**

### CODE REVIEW

**Review by victorisr — APPROVED** [review]

File: release-notes/9.0/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 | Date | Release |
 | :-- | :-- |
+| 2024/12/10 | [9.0.200 Preview SDK](./9.0.0/9.0.200-preview.md) |
```
Comment (campersau) [inline release-notes/9.0/README.md:17]: This link is not correct or the location of the `9.0.200-preview.md` file is wrong.

**Comment by Supermenwild** [comment]
[Command.txt](https://github.com/user-attachments/files/18142781/Command.txt)

**Review by Supermenwild — COMMENTED** [review]
200prv

**Review by Supermenwild — COMMENTED** [review]
gh pr checkout 9657

**Review by Supermenwild — COMMENTED** [review]
gh pr checkout 9662

**Pull request merged**